jina/jaml/__init__.py

```diff
@@ -494,8 +600,8 @@ def load_config(
         :class:`BaseExecutor`, :class:`BaseDriver` and all their subclasses.
 
         Support substitutions in YAML:
-            - Environment variables: `${{ENV.VAR}}` (recommended), ``${{VAR}}``.
-            - Context dict (``context``): ``${{VAR}}``(recommended).
+            - Environment variables: ``${{ ENV.VAR }}`` (recommended), ``$VAR`` (deprecated).
```

Comment:
If it's deprecated, why are we keeping it?

Reply:
General question. can be raised in slack or in an alignment. How long are we going to keep it around?

Reply:
We are trying to align with GitHub actions syntax, in which both work, but the first version is encouraged. Han told us to deprecate the second version, my guess is it will stick around for as long as it does within github actions.